start.sh

```diff
@@ -18,15 +18,101 @@ usage() {
     echo 'that command, for testing purposes.'
 }
 
+tritonConfigured() {
+
+    # is node-triton installed?
+    local triton_installed=0
+    command -v triton >/dev/null 2>&1 && triton_installed=1
+    if [ "0" = "$triton_installed" ]; then
+        echo
+        echo "Error!"
+        echo "The Triton CLI tool does not appear to be installed!"
+        echo
+        echo "Please visit:"
+        echo "https://www.joyent.com/blog/introducing-the-triton-command-line-tool"
+        echo "for installation instructions."
+        echo
+
+        sleep 3
+        command -v open >/dev/null 2>&1 && `open https://www.joyent.com/blog/introducing-the-triton-command-line-tool` || true
```

Comment:
I _knew_ you'd have thoughts on this. If I admit that I've never tried changing terminal colors, can I still be part of the cool kids club?